editors/gsecontrol/gse-control-element-editor.spec.ts

```diff
@@ -30,19 +30,22 @@ describe('GSEControl element editor component', () => {
 
   it('allows to change the GSEControl elements attributes', async () => {
     editor.gSEControlInputs![0].value = 'SomeNewName';
-    editor.gSEControlInputs![1].nullSwitch?.click();
```

Comment:
This might be a little difficult to follow but this particular switch click seemed not to be required, as `gse2` already had a description.

https://github.com/OpenEnergyTools/oscd-publisher/blob/52e7af1523c27c2ce1c5019a8618911e12516996/editors/gsecontrol/gse-control-element-editor.spec.ts#L16-L19

https://github.com/OpenEnergyTools/oscd-publisher/blob/52e7af1523c27c2ce1c5019a8618911e12516996/editors/gsecontrol/gseControl.testfiles.ts#L45

Reply:
Thx for the hint